src/scss/components/carousel.scss

```diff
@@ -2,7 +2,7 @@
   // The navbar has a 1px border on the bottom, so adding it to the margin
   margin-top: -($line-height-computed + 1);
   margin-bottom: $jumbotron-padding * 2;
-  border-bottom: 1px solid $carousel-border-color;
+  border-bottom: 1px solid $gray-300;
```

Comment:
I think we also need to set z-index on the carousel as it currently sits on top of the lower edge of the nav menu. If you set z-index to -2 then that seems to fix it.

Reply:
Actually z-index of -2 breaks the hover over on the nav buttons, so will need to investigate this.